pavelib/assets.py

```diff
@@ -56,17 +70,21 @@ def configure_paths():
         lms_css = theme_dir / "lms" / "static" / "css"
         if lms_sass.isdir():
             lms_css.mkdir_p()
-            THEME_SASS_DIRECTORIES.append(lms_sass)
+            THEME_SASS_DIRECTORIES.append(("lms/static/sass", theme_dir / "lms" / "static" / "css"))
```

Comment:
Nit: You can replace `theme_dir / "lms" / "static" / "css"` with `lms_css`, which is already defined a couple of lines above.